v2.0/releases/index.md

```diff
@@ -8,8 +8,26 @@ Use the version selector at the top-right of this page to view a different relea
 
 ## v2.0.0
 
+Notable Features:
+
+- Most of Calico's components have been ported from python to golang, including:
+  calicoctl, libcalico (as libcalico-go), calico-cni, libnetwork-plugin, most of
+  Felix, and parts of calico/node. The golang port allows for better alignment
+  with many of the projects we rely on (e.g. we now use the native APIs for etcd
+  and Prometheus directly).
+- calicoctl introduces a new
+  [resource-based UX model]({{site.baseurl}}/{{page.version}}/reference/calicoctl/commands).
+- Felix's golang rewrite adds major performance improvements particularly around resync / catch-up behavior.
+- Improved packaging that now vendors our dependencies; should avoid Python dependency issues such as incompatible urllib3/requests versions
+- Felix now shares more code with libcalico-go, the new golang Calico library.
+- Felix introduces a new dataplane driver API, which will allow for non iptables dataplanes in future
+- A new pluggable backend model has been introduced for Calico, removing the need for etcd by adding support for
```

Comment:
> removing the need for etcd

hmm wondering if this can be worded differently. Might sound a bit like we're phasing out etcd.